src/book/entities/Account.ts

```diff
@@ -185,34 +183,6 @@ function CheckAccountType(validationOptions?: v.ValidationOptions) {
   };
 }
 
-/**
- * Checks that investment accounts commodity is not a currency
- */
-function CheckInvestmentCommodity(validationOptions?: v.ValidationOptions) {
-  return function f(object: Account, propertyName: string) {
-    v.registerDecorator({
-      name: 'checkInvestmentCommodity',
-      target: object.constructor,
-      propertyName,
-      options: validationOptions,
-      validator: {
-        validate(fk_commodity: Commodity, args: v.ValidationArguments) {
-          const account = args.object as Account;
-          if (isInvestment(account)) {
-            return fk_commodity.namespace !== 'CURRENCY';
-          }
-
-          return true;
-        },
-
-        defaultMessage() {
-          return 'investment accounts cant have a currency as their commodity';
-        },
-      },
-    });
-  };
-}
```

Comment:
not anymore, robo advisors can be in the currency. You loose information because theoretically you should track all the share movements but that is too much in some cases